packages/menu-bar/test/menu-bar.test.js

```diff
@@ -326,8 +326,11 @@ describe('overflow button', () => {
   let menu, buttons, overflow;
 
   beforeEach(async () => {
-    menu = fixtureSync('<vaadin-menu-bar></vaadin-menu-bar>');
+    fixtureSync(
+      '<div><div style="width:100px;display:inline-block;"></div><vaadin-menu-bar style="display:inline-block;"></vaadin-menu-bar></div>',
+    );
 
+    menu = document.querySelector('vaadin-menu-bar');
     menu.style.width = '250px';
```

Comment:
Let's not change `display` property and instead place the menu-bar in a flex container.

This way the test setup would be more realistic (i.e. similar to what users tend to do).

```suggestion
  let wrapper, menu, buttons, overflow;

  beforeEach(async () => {
    wrapper = fixtureSync(`
      <div style="display: flex">
        <div style="width: 100px;"></div>
        <vaadin-menu-bar style="display:inline-block; width: 250px"></vaadin-menu-bar>
      </div>
    `);

    menu = wrapper.querySelector('vaadin-menu-bar');
```

Reply:
UPD: looks like this suggestion can't be applied via GitHub UI, so please consider editing the file manually.